en/admin/control.rst

```diff
@@ -43,7 +43,41 @@ One of the following can be specified in <command>:
 *   restart: restart a database server process.
 *   status: check status of a database server process. 
 
-Every command except **status** must have a database name as an argument. 
+| Every command can have a database name (**[database_name]**) as an argument to specify the database to execute the command.
+| If the database name is not specified in the **status** command, names of currently running database servers are displayed. see :ref:`control-cubrid-server-status`.  
```

Comment:
Would it be better if it changes as following? might make the sentence more clear. 

```suggestion
| If the database name is not specified in the **status** command, the currently running database servers' names are displayed. see :ref:`control-cubrid-server-status`.  
```

Reply:
Thank you